backend/core/database.py

```diff
@@ -148,9 +161,11 @@ def __init__(self, connection, db_conn, db_cursor):
             self.withdraw_queue = Queue(
                 'db-withdraw-update', exchange=self.update_exchange, routing_key='withdraw', durable=False, auto_delete=True, max_priority=1,
                 consumer_arguments={'x-priority': 1})
+
             self.hijack_queue = Queue(
-                'db-hijack-update', exchange=self.hijack_exchange, routing_key='update', durable=False, auto_delete=True, max_priority=1,
+                'db-hijack-update-{}'.format(uuid()), exchange=self.hijack_hashing, routing_key='1', durable=False, auto_delete=True, max_priority=1,
```

Comment:
What is the functionality of this change?

Reply:
they are different queues. if they had the same name it would have been the same queue